### PR TITLE
Create Into interface for Mat

### PR DESCRIPTION
**Pull request by AgustinCB**
As discussed in #74.

Some caveats: I wasn't able to mark the implementation or the function as `unsafe` without rustc compiling.

### CODE REVIEW

File: src/mat.rs [hunk 1]
```diff
@@ -10,11 +10,12 @@ use std::path::Path;
...
 use std::ops::{BitAnd, BitOr, BitXor, Not};
 
+/// The class `CMat` is used as a pointer to represent the Mat opencv structure
 #[derive(Clone, Copy, Debug)]
-pub(crate) enum CMat {}
+pub enum CMat {}
 
 impl CMat {
-    pub fn new() -> *mut CMat {
+    pub(crate) fn new() -> *mut CMat {
```
Comment (AgustinCB) [inline src/mat.rs]: I did this to ensure that you don't create `CMat`, but rather get it from a `Mat`. Debatable decision.

**Comment by Pzixel** [comment]
As I said, in Rust taking a pointer to any memory location is safe, it's unsafe when you're trying to deal with it. It's impossible to work with `CMat` in safe context, so people probably won't do it. Great job btw.

File: src/mat.rs (hunk 1 above)
Comment (Pzixel) [reply src/mat.rs]: I completely agree on that

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cv"
-version = "0.2.2"
+version = "0.2.1"
```
Comment (Pzixel) [inline Cargo.toml]: Why reverting? I have a strong feeling that every code change have to change a version.

File: tags [hunk 3]
```diff
@@ -0,0 +1,892 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```
Comment (Pzixel) [inline tags]: Please, do not commit autogenerated files.

**Pull request merged**